source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,86 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you paste a connection string into the |compass| connection form, by
+default, |compass-short| shows credentials in plaintext. To hide credentials in 
```

Comment:
[nit, editing my original suggestion 😅]

This might read slightly better as:

When you paste a connection string into the |compass| connection form, |compass-short| shows credentials in plaintext by default.

Reply:
Haha I was really debating between this and the original suggestion — couldn't tell which one read better when juggling with it. 😅